tcms/testruns/tests/test_admin.py

```diff
@@ -91,3 +96,48 @@ def test_changelist_view_icon_column(self):
             """
             ),
         )
+
+    @parameterized.expand(
+        [
+            ("positive_tests", positive),
```

Comment:
You can't quite reference the test data like that!  If those are class attributes (or object attributes created in setUp) then you need to prefix them with either `self.positive` or `TestTestExecutionStatusAdmin.positive`. None of which will work.

`self` isn't defined outside of a method (which is where the decorator needs to be written). 

Using the class name to access class attributes for this decorator while the class is till being initialized will lead to circular references to the class name and not work either.

The only option is for the variables holding test data to become global variables. And while we're on the same topic we don't need variables b/c you can pass the querysets directly as arguments to `@parameterized.expand`.